x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -328,8 +328,8 @@ export const removeDataProvider = () => {
 };
 
 export const createNewTimeline = () => {
-  cy.get(NEW_TIMELINE_ACTION).should('be.visible').trigger('click');
-  cy.get(CREATE_NEW_TIMELINE).eq(0).should('be.visible').click({ force: true });
+  cy.get(NEW_TIMELINE_ACTION).should('be.visible').click();
+  cy.get(CREATE_NEW_TIMELINE).first().should('be.visible').click();
```

Comment:
same here, in theory we shouldn't need the `should('be.visible')` here as the click action from Cypress should scroll to the element and ensure the element is not hidden before clicking (see [Cypress documentation](https://docs.cypress.io/guides/core-concepts/interacting-with-elements))

Reply:
https://github.com/elastic/kibana/pull/173276/commits/10a818d6da4fa2e0c0ae508c7960094d09891f12  👍